### ISSUE TITLE
RemoteConfigProperty does not appear when using via Cocoapods

### ISSUE THREAD

**Issue by masarusanjp** [firebase/firebase-ios-sdk#10371]
### Description

`RemoteConfigProperty` introduced in the [release notes](https://firebase.google.com/support/release-notes/ios#10.0.0)  does not seem to be available.

I tried to add `FirebaseRemoteConfigSwift` in Podfile and use it, but the definition for `RemoteConfigProperty` does not seem to exist.

Sorry if I am mistaken and `RemoteConfigProperty` is not yet available.

I think it may be due to the fact that the source_files designation on the podspec in Cocoapods/Specs is `"FirebaseRemoteConfigSwift/Sources/*.swift"`.
                                  `https://github.com/CocoaPods/Specs/blob/master/Specs/e/b/d/FirebaseRemoteConfigSwift/10.0.0/FirebaseRemoteConfigSwift.podspec.json#L25-L27

In https://github.com/firebase/firebase-ios-sdk/blob/CocoaPods-10.0.0/FirebaseRemoteConfigSwift.podspec#L37-L39, the `' FirebaseRemoteConfigSwift/Sources/**/*.swift', so it's odd why the difference.

### Reproducing the issue

- Cerate new xcode project
- write Podfile.
```
pod 'FirebaseRemoteConfigSwift', '>= 10.0.0'
```

### Firebase SDK Version

10.0.0

### Xcode Version

14.0.1

### Installation Method

CocoaPods

### Firebase Product(s)

Remote Config

### Targeted Platforms

iOS

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

PODS:
  - FirebaseABTesting (10.0.0):
    - FirebaseCore (~> 10.0)
  - FirebaseCore (10.0.0):
    - FirebaseCoreInternal (~> 10.0)
    - GoogleUtilities/Environment (~> 7.8)
    - GoogleUtilities/Logger (~> 7.8)
  - FirebaseCoreInternal (10.0.0):
    - "GoogleUtilities/NSData+zlib (~> 7.8)"
  - FirebaseInstallations (10.0.0):
    - FirebaseCore (~> 10.0)
    - GoogleUtilities/Environment (~> 7.8)
    - GoogleUtilities/UserDefaults (~> 7.8)
    - PromisesObjC (~> 2.1)
  - FirebaseRemoteConfig (10.0.0):
    - FirebaseABTesting (~> 10.0)
    - FirebaseCore (~> 10.0)
    - FirebaseInstallations (~> 10.0)
    - GoogleUtilities/Environment (~> 7.8)
    - "GoogleUtilities/NSData+zlib (~> 7.8)"
  - FirebaseRemoteConfigSwift (10.0.0):
    - FirebaseRemoteConfig (~> 10.0)
    - FirebaseSharedSwift (~> 10.0)
  - FirebaseSharedSwift (10.0.0)
  - GoogleUtilities/Environment (7.8.0):
    - PromisesObjC (< 3.0, >= 1.2)
  - GoogleUtilities/Logger (7.8.0):
    - GoogleUtilities/Environment
  - "GoogleUtilities/NSData+zlib (7.8.0)"
  - GoogleUtilities/UserDefaults (7.8.0):
    - GoogleUtilities/Logger
  - PromisesObjC (2.1.1)

DEPENDENCIES:
  - FirebaseRemoteConfigSwift (>= 10.0.0)

SPEC REPOS:
  trunk:
    - FirebaseABTesting
    - FirebaseCore
    - FirebaseCoreInternal
    - FirebaseInstallations
    - FirebaseRemoteConfig
    - FirebaseRemoteConfigSwift
    - FirebaseSharedSwift
    - GoogleUtilities
    - PromisesObjC

SPEC CHECKSUMS:
  FirebaseABTesting: 5a08a4d3060b7fb5638f60698bc2cb57996bcc73
  FirebaseCore: 97f48a3a567a72b8d4daa0f03c3aadb78df4e995
  FirebaseCoreInternal: 5eb3960335da5ea30115d57d39db6988c4ad06f3
  FirebaseInstallations: 7f1c9ae6bd9df6abe9c74124b38fa8740aba5df4
  FirebaseRemoteConfig: e4431ddba74ddf705e2aabd7d356a23d5b802853
  FirebaseRemoteConfigSwift: b3bc06b3f0ee8f481d8ea700edc1334a01dab29d
  FirebaseSharedSwift: eae0978bbb46dbfc21975e88a79654f614822ce4
  GoogleUtilities: 1d20a6ad97ef46f67bbdec158ce00563a671ebb7
  PromisesObjC: ab77feca74fa2823e7af4249b8326368e61014cb

PODFILE CHECKSUM: da6616e576126aaa6c4310e9e35f8a7797ce3149

COCOAPODS: 1.11.2

```

</details>

**Comment by rizafran** [comment]
Thanks for reporting, @masarusanjp. Per checking, it seems that `RemoteConfigProperty` doesn't exist in the latest version in `CocoaPods` but it's available in `SPM`. I'll inform the engineers about this.

**Comment by chliangGoogle** [comment]
Thank you for detecting the issue and quickly reports to us @masarusanjp. The fix should go out to Firebase 10.1.0.
